draft-reddy-cose-jose-pqc-hybrid-hpke.md

```diff
@@ -113,6 +113,10 @@ For readability the algorithm ciphersuites labels are built according to the fol
 
 The HPKE PQ/T hybrid ciphersuites for JOSE and COSE are defined in {{IANA}}. Note that the PQ/T Hybrid KEM in HPKE is not an authenticated KEM. The HPKE Base mode can only be supported with the PQ/T Hybrid KEM.
 
+# Key Management
```

Comment:
This should presumably be a subsection of Security Considerations instead of being its own top-level section.

Reply:
Yes, moved text.